core/src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20170302_add_method_config_version_to_unique_index.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet id="method_config_drop_index" author="ansingh" logicalFilePath="dummy">
+        <sql stripComments="true">
+            DROP INDEX IDX_CONFIG ON METHOD_CONFIG;
+        </sql>
+    </changeSet>
+
+    <changeSet id="method_config_create_index" author="ansingh" logicalFilePath="dummy">
+        <sql stripComments="true">
+            CREATE INDEX IDX_CONFIG ON METHOD_CONFIG (NAMESPACE, NAME, WORKSPACE_ID, METHOD_CONFIG_VERSION);
```

Comment:
a few things:

- you'll also want to update the index that's defined in `MethodConfigurationTable` in `MethodConfigurationComponent.scala`

- this new index is no longer unique. `CREATE UNIQUE INDEX ...` should work

- ordering the columns in the index as `WORKSPACE_ID, NAMESPACE, NAME, METHOD_CONFIG_VERSION` would allow for the index to be used by more queries. the specific one that i'm thinking about is listing all method configs for a workspace- which wouldn't be able to use the index in it's current order because `WORKSPACE_ID` is not a prefix of the index. the change in column order is what likely caused the results of your tests to change